project/V.scala

```diff
@@ -65,13 +65,14 @@ object V {
   // Scala 2
   def deprecatedScala2Versions = Seq(
     scala211,
-    "2.12.9",
     "2.12.10",
     "2.12.11",
-    "2.13.1",
-    "2.13.2",
+    "2.12.12",
+    "2.12.13",
     "2.13.3",
     "2.13.4",
+    "2.13.5",
+    "2.13.6",
   )
 
   def nonDeprecatedScala2Versions = Seq(
```

Comment:
Maybe we should just come up with some random rule to follow like:

- Latest 3 versions are non deprecated
- Then another 2 are deprecated

So we'd then have support for 5 versions each, which to be honest should be enough. Since bumping from like 2.12.10 -> 2.12.whatever shouldn't be hard.

Reply:
That makes sense the 3 + 2 formula. Should we add it to the release procedure to check these? Or maybe remove after the release?

Reply:
Yea maybe after the release make sense.